main.py

```diff
@@ -1,10 +1,59 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for user, info in data.items():
+        nickname = user
+        email = info.get("email")
+        bio = info.get("bio")
+
+        if info["guild"]:
+            guild_name = (info["guild"]["name"]
+                          if info["guild"]["name"] else None)
+            guild_description = (info["guild"]["description"]
+                                 if info["guild"]["description"] else None)
+
+            if not Guild.objects.filter(name=guild_name).exists():
+                guild_link = Guild.objects.create(
+                    name=guild_name,
+                    description=guild_description
+                )
```

Comment:
This will not affect the number of database queries but may improve readability. For this, use the get_or_create() method. Just try it )